packages/react/src/provider.tsx

```diff
@@ -72,6 +73,14 @@ export const LogtoProvider = ({ logtoConfig, children }: LogtoProviderProperties
       try {
         await logtoClient.handleCallback(uri);
         dispatch({ type: 'HANDLE_CALLBACK_SUCCESS', payload: { claims: logtoClient.getClaims() } });
+        if (window.location.href.includes('?')) {
+          const { url: baseUrl, query: queryObject } = queryString.parseUrl(window.location.href);
+          if (baseUrl && Object.keys(queryObject).length > 0) {
+            const { code, state, ...filteredEntries } = queryObject;
+            const newUrl = queryString.stringifyUrl({ url: baseUrl, query: filteredEntries });
```

Comment:
The if statement and querystring parse & stringify are unnacessary:
1. we know that current `location.href` a valid callback url with code and state, or we won't call this function (`handleCallback`)
2. the `redirectUri` is already set and we know exactly what it is, no need to recover query entries.
cc @simeng-li

Reply:
make sense